Shared/Samples/Manage operational layers/ManageOperationalLayersView.swift

```diff
@@ -26,128 +26,119 @@ struct ManageOperationalLayersView: View {
         return map
     }()
     
-    /// A Boolean value indicating whether to show the manage layers sheet.
-    @State private var isShowingSheet = false
+    /// The operational layers for the sample.
+    @State private var operationalLayers: [Layer] = [
+        ArcGISMapImageLayer(url: .worldElevations),
+        ArcGISMapImageLayer(url: .censusTiles)
+    ]
     
-    /// The error shown in the error alert.
-    @State private var error: Error?
+    /// A Boolean value indicating whether the manage layers view is presented.
+    @State private var manageLayersIsPresented = false
     
     var body: some View {
         MapView(map: map)
             .task {
-                do {
-                    // Add layers from urls.
-                    let elevationImageLayer = ArcGISMapImageLayer(url: .worldElevations)
-                    try await elevationImageLayer.load()
-                    
-                    let censusTiledLayer = ArcGISMapImageLayer(url: .censusTiles)
-                    try await censusTiledLayer.load()
-                    
-                    map.addOperationalLayers([elevationImageLayer, censusTiledLayer])
-                } catch {
-                    self.error = error
-                }
+                // Loads and adds the operational layers to the map.
+                await operationalLayers.load()
+                map.addOperationalLayers(operationalLayers)
             }
             .toolbar {
                 ToolbarItem(placement: .bottomBar) {
                     Button("Manage Layers") {
-                        isShowingSheet = true
+                        manageLayersIsPresented = true
                     }
-                    .popover(isPresented: $isShowingSheet) {
-                        ManageLayersSheetView(map: map)
-                            .presentationDetents([.fraction(0.5)])
-                            .frame(idealWidth: 320, idealHeight: 360)
+                    .popover(isPresented: $manageLayersIsPresented) {
+                        NavigationStack {
+                            ManageLayersView(map: map, layers: operationalLayers)
+                                .navigationTitle("Manage Layers")
+                                .navigationBarTitleDisplayMode(.inline)
+                                .toolbar {
+                                    ToolbarItem(placement: .confirmationAction) {
+                                        Button("Done") { manageLayersIsPresented = false }
+                                    }
+                                }
```

Comment:
These kinds of view modifiers are usually on the view itself rather than on the parent view as they are intrinsic to how the view should be displayed and how it should operate. The layers view should use the `dismiss` environment value to dismiss itself when the Done button is tapped/clicked.